pkg/discovery/utils.go

```diff
@@ -70,3 +80,96 @@ func compareRESTDiscoverySources(ds1, ds2 configtypes.PluginDiscovery) bool {
 		ds1.REST.BasePath == ds2.REST.BasePath &&
 		ds1.REST.Endpoint == ds2.REST.Endpoint
 }
+
+func getDiscoverySourceNameAndURL(source configtypes.PluginDiscovery) (string, string, error) {
+	var name string
+	var url string
+	switch {
+	case source.OCI != nil:
```

Comment:
We only support OCI, so you could actually use that without the others below

Reply:
Updated the switch to only use oci and default